src/info/updates/archive.md

```diff
@@ -8,11 +8,22 @@ meta:
 
 # Archived updates from the FAC
 
-The FAC is committed to working transparently and openly. In addition to our regular updates, you can find historical updates below.
-
-## Week of October 30, 2023
-
-We launched search and are ready for users to begin searching single audit reports. This initial iteration of search leverages six different filters for sorting data.
+The FAC is committed to working transparently and openly. In addition to <a href=https://www.fac.gov/info/updates/>our regular updates</a>, you can find historical updates below.
+
+<div class="usa-accordion usa-accordion--bordered">
+  <h4 class="usa-accordion__heading">
+    <button
+      type="button"
+      class="usa-accordion__button"
+      aria-expanded="true"
+      aria-controls="b-a1"
+    >
+      Week of October 30, 2023
+    </button>
+  </h4>
+  <div id="b-a1" class="usa-accordion__content usa-prose">
+    <p>
+      We launched search and are ready for users to begin searching single audit reports. This initial iteration of search leverages six different filters for sorting data.
```

Comment:
```suggestion

      We launched search and are ready for users to begin searching single audit reports. This initial iteration of search leverages six different filters for sorting data.
```

Does this paragraph need a closing `</p>` paragraph tag? Or maybe it doesn't need the _opening_ `<p>` tag?